cylc/flow/scripts/cylc.py

```diff
@@ -430,24 +433,28 @@ def cli_help():
     # modern terminal)
     from colorama import init as color_init
     color_init(autoreset=True, strip=False)
+    commands = [
+        'clean',
+        'hold',
+        'install',
```

Comment:
Is reinstall important enough to be added here?

Reply:
Fair point, subjective, I didn't add `reload` which was silly, if we are having `reinstall` we need `reload` too.

Reply:
(That change has been merged)